Tools/eslint-config-cesium/browser.js

```diff
@@ -1,13 +1,16 @@
-{
-    "extends": "../../.eslintrc",
+/*eslint-env node*/
+'use strict';
+
+module.exports = {
+    "extends": "eslint-config-cesium",
     "globals": {
```

Comment:
These globals should stay in Sandcastle only, they are not common to all browsers.  `plugins` can stay here.